api/src/main/java/com/community/api/endpoint/avisoft/controller/Qualification/QualificationDetailsController.java

```diff
@@ -92,19 +125,28 @@ public ResponseEntity<?> deleteQualificationDetailById(@PathVariable Long custom
         }
         catch (Exception e)
         {
+            exceptionHandling.handleException(e);
             return ResponseService.generateErrorResponse("Something went wrong",HttpStatus.BAD_REQUEST);
         }
     }
 
     @PutMapping("/update/{customCustomerId}/{qualificationDetailId}")
-    public ResponseEntity<?> updateQualificationDetailById(@PathVariable Long customCustomerId, @PathVariable Long qualificationDetailId, @Valid @RequestBody UpdateQualificationDto qualification) throws EntityDoesNotExistsException, EntityAlreadyExistsException, ExaminationDoesNotExistsException, CustomerDoesNotExistsException {
+    public ResponseEntity<?> updateQualificationDetailById(@PathVariable Long customCustomerId, @PathVariable Long qualificationDetailId, @Valid @RequestBody UpdateQualificationDto qualification,@RequestHeader(value = "Authorization") String authHeader) throws EntityDoesNotExistsException, EntityAlreadyExistsException, ExaminationDoesNotExistsException, CustomerDoesNotExistsException {
```

Comment:
here if someone enters string
give NumberFormatException exception

use this
Long id = Long.valueOf(customerId);

catch (NumberFormatException e) {
    return ResponseService.generateErrorResponse("Invalid customerId: expected a Long", HttpStatus.BAD_REQUEST);
}